src/deviation.jl

```diff
@@ -137,7 +137,7 @@ end
 
 Return the mean absolute deviation between two arrays: `mean(abs(a - b))`.
 """
-meanad{T<:Number}(a::AbstractArray{T}, b::AbstractArray{T}) = L1dist(a, b) / length(a)
+meanad(a::AbstractArray{T}, b::AbstractArray{T}) where {T<:Number} = L1dist(a, b) / length(a)
```

Comment:
That's actually a larger issue with this package: the lines are looooooooong. I just did the quick and dirty syntax rewrite in this PR but we should consider going through and doing a stylistic cleanup in a separate PR.